app/addons/fauxton/navigation/stores.js

```diff
@@ -94,8 +107,7 @@ Stores.NavBarStore = FauxtonAPI.Store.extend({
   },
 
   toggleMenu () {
-    app.utils.localStorageSet(FauxtonAPI.constants.LOCAL_STORAGE.SIDEBAR_MINIMIZED,
-                              !this.isMinimized());
+    this._isMinimized = !this._isMinimized;
```

Comment:
This means we are no longer keeping state of what the user preferred in terms of minimized or open navbar?